project/Build.scala

```diff
@@ -507,4 +512,16 @@ object Build extends sbt.Build {
         )
       )
   )
+
+  lazy val pipeline = Project(
+    id = "gearpump-experiments-pipeline",
+    base = file("experiments/pipeline"),
+    settings = commonSettings ++ myAssemblySettings ++
+      Seq(
+        libraryDependencies ++= Seq(
+          "org.apache.hbase" % "hbase-client" % clouderaHBaseVersion,
+          "org.apache.hbase" % "hbase-common" % clouderaHBaseVersion
```

Comment:
if i move to examples/streaming/kafka_hbase_pipeline then I will need to depend on hbase under experiments. Perhaps both should be moved? Although HBaseSink isn't really a streaming example.... Maybe I move HBaseSink into kafka_hbase_pipeline and rename HBase to HBaseConsumer. This way we have 
```
KafkaProducer (uses KafkaSource)
HBaseConsumer (uses HBaseSink)
```
Or we could have a external_hbase similar to external_kafka